index.html

```diff
@@ -1661,6 +1671,17 @@ <h2 id="AudioContext">
           use cases, only a single <a><code>AudioContext</code></a> is used per
           document.
         </p>
+        <p>
+          An <a><code>AudioContext</code></a> is said to be <dfn>allowed to
+          start</dfn> if the user agent and the system allow sound output in
```

Comment:
"sound" -> "audio" ?

I think this sentence needs to be expanded a bit. It's kind of described later what happens, but a little more information here would be a useful introduction.